docs/source/geopm_agent_ffnet.7.rst

```diff
@@ -72,6 +74,35 @@ Policy Requirements
 
 The ``PERF_ENERGY_BIAS`` must be between 0 and 1.
 
+Required JSON Files
+--------------------
+
+To generate the neural network and frequency recommendation files, the following procedure
+can be followed. Note that all python files referenced can be found in the directory
+``$GEOPM_SOURCE/integration/experiment/ffnet`` unless otherwise specified.
+
+# Run the ``neural_net_sweep.py`` on microbenchmarks of interest. Some microbenchmarks
+  that are supported in the integration infrastructure and have been demonstrated to 
+  be useful include:
+
+  * [CPU] Arithmetic Intensity Benchmark
+  * [CPU] geopmbench
+  * [GPU] parres dgemm
+  * [GPU] parres stream
+
+# Run ``gen_hdf_from_fsweep.py`` to generate HDF files from the frequency sweep reports
+  and traces
+
+# Run ``gen_neural_net.py`` to generate the neural net(s). CPU/GPU nets will be generated
```

Comment:
Similar comment about cross-referencing the detailed doc, with an additional comment:

If I'm a more interested in just seeing things work and I don't care as much about the details, then I might just blindly following the steps listed here. One case that I see breaks here is if `pytorch` isn't already installed. Are there other pre-requisites that are needed or is that all? If that's all, maybe just note it here to simplify things for the the copy/paste user. Otherwise, maybe we'd benefit from an additional setup script?

Reply:
I think it's just pytorch, but I would love someone to try it and let me know if I'm forgetting anything!